### PR TITLE
Updated documentation of stats.summary

### PR DESCRIPTION
**Pull request by denadai2**
Nothing special, I enhanced the documentation :)

### CODE REVIEW

**Review by springcoil — APPROVED** [review]
Well done :)

**Comment by springcoil** [comment]
We'll merge this in when it passes. Welcome @denadai2 !

File: pymc3/stats.py [hunk 1]
```diff
@@ -58,21 +58,35 @@ def wrapped_f(pymc3_obj, *args, **kwargs):
...
 def autocov(x, lag=1):
-    """
-    Sample autocovariance at specified lag.
-    The autocovariance is a 2x2 matrix with the variances of
-    x[:-lag] and x[lag:] in the diagonal and the autocovariance
-    on the off-diagonal.
-    """
+    """Sample autocovariance at specified lag.
+
+        Parameters
```
Comment (twiecki) [inline pymc3/stats.py]: De-indent.

File: pymc3/stats.py [hunk 2]
```diff
@@ -58,21 +58,35 @@ def wrapped_f(pymc3_obj, *args, **kwargs):
...
 
 @statfunc
 def autocov(x, lag=1):
-    """
-    Sample autocovariance at specified lag.
-    The autocovariance is a 2x2 matrix with the variances of
-    x[:-lag] and x[lag:] in the diagonal and the autocovariance
-    on the off-diagonal.
-    """
+    """Sample autocovariance at specified lag.
```
Comment (twiecki) [inline pymc3/stats.py]: line break to have """ on a single line.

Comment (denadai2) [reply pymc3/stats.py]: Do you want to have

    """
    Sample autocovariance at specified lag.

?

Comment (denadai2) [reply pymc3/stats.py]: As far as I see documentation should be """Comment
ref: http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html

Comment (twiecki) [reply pymc3/stats.py]: You're right, let's leave this.

File: pymc3/stats.py (hunk 1 above)
Comment (twiecki) [reply pymc3/stats.py]: This should still be fixed.

**Comment by denadai2** [comment]
It should be ok now :))

**Pull request merged**